.github/workflows/test.yml

```diff
@@ -0,0 +1,24 @@
+name: Test and update
+
+on:
+  schedule:
+    - cron: "37 6 * * *"
+  workflow_dispatch: {}
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: github.repository == 'williamkapke/node-compat-table'
+
+    permissions:
+      contents: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+
+      - run: npm install
```

Comment:
```suggestion
      - uses: ljharb/node/install@main
```

Reply:
Testing this out on my fork, it looks like nvm doesn't make it into PATH? https://github.com/jakebailey/node-compat-table/actions/runs/4482188832/jobs/7879898199

Reply:
hm, it should, since i have 300 repos running `npm test` on the next step.

Reply:
`nvm`, not `npm`; but it's certainly confusing because my skimming of the action implies it should.

Reply:
ah, it might need `. $NVM_DIR/nvm.sh` inside the script to make the sourced shell function available.

Reply:
Oh, right, since nvm is not just on PATH, it's shell magic. (Not my favorite but, whatever)